talisker/sentry.py

```diff
@@ -141,14 +153,18 @@ def record_log_breadcrumb(record):
 
         def processor(data):
             metadata = {
-                'path': record.pathname,
-                'lineno': record.lineno,
+                'location': '{}:{}'.format(record.pathname, record.lineno)
             }
-            if hasattr(record, 'func'):
-                metadata['func'] = record.func
-            metadata.update(getattr(record, '_structured', {}))
+            funcName = getattr(record, 'funcName', None)
+            if funcName:
+                metadata['location'] += ':' + record.funcName + '()'
```

Comment:
I feel the file:line:func format is pretty common and recognisable and so the '()' postfix is not needed (I haven't seen that syntax used before).
Also perhaps there is an argument for having the second colon even if you can't get the function name as it might make it easier to parse by eye or in scripts.

Reply:
Good call, will do